modules/bbp-simulation/src/test/resources/data/bbp/core/entity/v0.1.0/all-fields.json

```diff
@@ -8,9 +8,41 @@
   ],
   "@id": "{{base}}/data/bbp/core/entity/v0.1.0/20e68e2d-47fe-47c1-bc5f-195d914d00ae",
   "name": "Edge property SSCx O1.v6a",
-  "distribution": [
+  "hadMember": [
     {
-      "downloadURL": "file:///projectpath/circuits/O1.v6a/20171206/ncsFunctionalAllRecipePathways/take the nrn files"
+      "distribution": [
+        {
+          "downloadURL": "file:///projectpath/circuits/O1.v6a/20171206/ncsFunctionalAllRecipePathways/nrn.h5"
+        }
+      ]
+    },
+    {
+      "distribution": [
+        {
+          "downloadURL": "file:///projectpath/circuits/O1.v6a/20171206/ncsFunctionalAllRecipePathways/nrn_efferent.h5"
```

Comment:
Could this be changed to "distribution": [{"downloadURL": "ex1"}, {"downloadURL": "ex2"}, ...]

Reply:
No because data attached to an entity should be published in a unique distribution.